### PR TITLE
Bulk create using json

### PR DESCRIPTION
**Pull request by Ali-D-Akbar**

### CODE REVIEW

File: bulk_books.JSON [hunk 1]
```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "model": "books.book",
+    "pk": 10,
```
Comment (asamolion) [inline bulk_books.JSON]: Let's say we do not want to supply the `id` or `model`, what will happen in that case?

File: userauth/admin.py [hunk 2]
```diff
@@ -17,9 +17,14 @@ class ProfileInline(admin.StackedInline):
 
 class CustomUserAdmin(UserAdmin):
     inlines = (ProfileInline, )
-    list_display = ('username', 'email', 'first_name', 'last_name', 'get_gender', 'get_phone_number')
+    list_display = ('get_profile_id', 'id', 'username', 'email', 'get_gender', 'get_phone_number')
```
Comment (asamolion) [inline userauth/admin.py]: Why are we using profile id instead of user id?

Comment (Ali-D-Akbar) [reply userauth/admin.py]: I'm using both user and profile id just to see the data in Admin Panel

File: bulk_books.JSON (hunk 1 above)
Comment (Ali-D-Akbar) [reply bulk_books.JSON]: it'll work without "pk": *id*
but it'll not work without "model": "books.book"

Comment (asamolion) [reply bulk_books.JSON]: Implement it such that the `model` key is not required.

File: userauth/admin.py (hunk 2 above)
Comment (asamolion) [reply userauth/admin.py]: Alrightey.

File: bulk_books.JSON (hunk 1 above)
Comment (Ali-D-Akbar) [reply bulk_books.JSON]: done

**Pull request merged**